schema.graphql

```diff
@@ -31,19 +49,33 @@ enum EscrowState {
 }
 
 type Escrow @entity {
+  "hash transaction of the escrow"
   id: ID!
+  "network contract address"
   contractAddress: Bytes!
+  "identifies a recipient and their payment account"
```

Comment:
I'm not sure this is correct. I'll have to double check.

Reply:
I double-checked, and this should be "payment proxy contract address"

Reply:
@PaulieB14 Just pinging since Github hides this one. This one still needs to fixed too.

Reply:
Revised to corrected